pkgs/development/libraries/mkldnn/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, cmake }:
+
+stdenv.mkDerivation rec {
+  pname = "mkldnn";
+  version = "1.0.2";
+
+  src = fetchFromGitHub {
+    owner = "intel";
+    repo = "mkl-dnn";
+    rev = "v${version}";
+    sha256 = "0lf14a4j511pydyz78wwbl6hifbq86m3dy119iss9cvz8zzrdyvw";
+  };
+
+  patches = [ ./bash-to-sh.patch ];
+
+  nativeBuildInputs = [ cmake ];
+
+  doCheck = true;
+  checkPhase = "ctest";
+
+  meta = with stdenv.lib; {
+    description = "Deep Neural Network Library (DNNL)";
+    homepage = "https://intel.github.io/mkl-dnn/dev_guide_transition_to_dnnl.html";
+    licence = licenses.asl20;
+    platforms = platforms.all;
+    maintainers = [];
```

Comment:
To be honest I only did this because there was an issue to add it, I have never used the software and never will so I'm not sure if I would be a good maintainer for it.

That being said, I am happy to add myself if it is a choice between that and no maintainer

Reply:
yea, not sure what would be preferable